css-brochure-form/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div>
+      <h4>LEARN MORE</h4>
+    </div>
+
+    <div>
+      <form action="submit">
+        <div class="field">
+          <label for="companyEmail">COMPANY EMAIL</label>
+          <div>
+            <input type="text" value="sally@company.co" />
+          </div>
+        </div>
+
+        <div class="field">
+          <label for="name">NAME</label>
+          <div>
+            <input type="text" value="Sally Smith" />
+          </div>
+        </div>
+
+        <div class="field">
+          <label for="whereHiring">WHERE ARE YOU HIRING?</label>
+          <div>
+            <input type="text" value="City, Country" />
+          </div>
+        </div>
+
+        <div class="field">
+          <div>
+            <label for="howManyEngineers"
+              >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+            >
+          </div>
+          <select>
+            <option>1-2 Engineers</option>
+          </select>
+        </div>
```

Comment:
Same thing here with the `label` and `for` attribute.

Also if it is a `select`, should it only have one `option`? Even if you don't see it I would recommend adding 2 more options here. Also, what attribute should an `option` have so we know which was selected?